prow/prowjobs/GoogleCloudPlatform/gcsfuse/gcsfuse-pytorch-dino-periodic.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+- name: gcsfuse-pytorch-dino-periodic
+  cluster: gcsfuse-prow-test
```

Comment:
> {"component":"checkconfig","file":"k8s.io/test-infra/prow/cmd/checkconfig/main.go:91","func":"main.reportWarning","level":"warning","msg":"invalid periodic job: job configuration for \"gcsfuse-pytorch-dino-periodic\" specifies unknown 'cluster' value \"gcsfuse-prow-test\"","severity":"warning","time":"2023-05-18T19:00:49Z"}

Is this a new cluster? Prow doesn't seem to recognize it. Can you point to where it was configured?

Reply:
Yes, this is a new cluster. I just ran https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/oss/create-build-cluster.sh. Is there any other configurations that I need to do ? If yes, please point to it.

> Prow jobs in this repository must contain TestGrid annotations or be explicitly opted-out. Try removing the commit that deletes the TestGrid annotations.

I removed the TestGrid annotations because that was giving me error - "No dashboard found". Do I have to first create the TestGrid dashboard ? If yes, is it true that I just need to add an entry here; https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/testgrid/config.yaml ?